sdk/modelsrepository/Azure.Iot.ModelsRepository/src/ModelRepositoryConstants.cs

```diff
@@ -8,6 +8,8 @@ internal static class ModelRepositoryConstants
         // Set EventSource name to package name replacing '.' with '-'
         public const string ModelRepositoryEventSourceName = "Azure-Iot-ModelsRepository";
 
+        public const string DefaultModelRepository = "https://devicemodels.azure.com";
```

Comment:
If we have Model in the name I think it should be plural like `DefaultModelsRepository` ...both the repo github project and the global service url refer to the plural models: https://github.com/Azure/iot-plugandplay-models ,  https://devicemodels.azure.com/

Reply:
Agreed. Changing.